lib/rules/headers/copyright.js

```diff
@@ -41,7 +41,7 @@ exports.check = function (sr, done) {
         ,   "http://www.csail.mit.edu/":                                        "MIT"
         ,   "http://www.ercim.eu/":                                             "ERCIM"
         ,   "http://www.keio.ac.jp/":                                           "Keio"
-        ,   "http://ev.buaa.edu.cn/":                                           "Beihang"
+        ,   "https://ev.buaa.edu.cn/":                                           "Beihang"
```

Comment:
nitpick: please align the word "Beihang" with the others

Reply:
It seems the Beihang website is the only https website here?

Reply:
@xfq it should now be fixed